spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalTaskLauncher.java

```diff
@@ -177,6 +185,23 @@ public RuntimeEnvironmentInfo environmentInfo() {
 		return super.createRuntimeEnvironmentInfo(TaskLauncher.class, this.getClass());
 	}
 
+	@Override
+	public int getMaximumConcurrentTasks() {
+		return getLocalDeployerProperties().getMaximumConcurrentTasks();
+	}
+
+
+	@Override
+	public int getRunningTaskExecutionCount() {
+		return
+			new Long(running.entrySet().stream().filter( e-> e.getValue().getProcess().isAlive()).count()).intValue();
```

Comment:
It definitely feels a bit weird to return Long when signature has int.

Reply:
The compiler would agree, must return an int.  Using `long Stream.count()`, so I'm converting it to int using `Long.intValue()`

Reply:
Ah yes, I missed to follow all the brackets.

Reply:
It's my understanding that it is considered bad practice to use streams where a simple `for` loop can be used given the performance implications (basically, streams are slower and more heavy weight).  Ollie mentioned it in this thread as an example: https://twitter.com/odrotbohm/status/1056866589763690497